docs/fundamentals/databases.md

```diff
@@ -0,0 +1,58 @@
+
+Geth also tracks some basic metrics relating to the ancients database that can be monitored:
+
+| metric       | function                                                   |
+| ------------ | ---------------------------------------------------------- |
+| `readMeter`  | Meter for measuring the effective amount of data read      |
+| `writeMeter` | Meter for measuring the effective amount of data written   |
+| `sizeGauge`  | Gauge for tracking the combined size of all freezer tables |
+
+
+The ancients data is saved entirely separately from the fast-access recent data, meaning it can be stored in a  different location. The default location for the ancient chain segments is inside the `chaindata` directory, which is inside `datadir`, but it can be defined by passing `--datadir.ancient <path>` to Geth on startup. This path can be to a slow HDD disk, permitting people to run Ethereum nodes without requiring a huge SSD. The ancient can also be moved later by manually copying the directory to a new location and then starting Geth passing the new path to `--datadir.ancient`.
+
+
+## Using the freezer
+
+If Geth cannot find the freezer, either because the database was deleted or because Geth received an incorrect path, then the node becomes unstable. It is explicitly forbidden to start Geth with an invalid path to the freezer. However, if the state database goes missing it can be rebuilt "on-the-fly" by Geth. In this case, Geth rebuilds all its indices based upon data from the freezer and then does a snap-sync on top of it to rebuild the missing state data. In summary - Geth can rebuild the recent past from the ancient past but not vice-versa.
+
+This can be used to deliberately clean up a node. Passing `--datadir --removedb` starts the process. Geth will ask whether it should delete the state database, the ancient database and the LES database. Choosing to delete the state database onyl leaves the block bodies, receipts, headers etc intact in the freezer, meaning rebuilding the state will not include re-downloading ~400GB of data from the network. Geth will then rebuild the state from the freezer reusing that existing block and receipt data. In doing so, unused data and accumulated junk data will be pruned from the state database. This process can take an hour or more.
```

Comment:
onyl -> only

Reply:
```suggestion
This can be used to deliberately clean up a node. Passing `--datadir --removedb` starts the process. Geth will ask whether it should delete the state database, the ancient database and the LES database. Choosing to delete the state database only leaves the block bodies, receipts, headers etc intact in the freezer, meaning rebuilding the state will not include re-downloading ~400GB of data from the network. Geth will then rebuild the state from the freezer reusing that existing block and receipt data. In doing so, unused data and accumulated junk data will be pruned from the state database. This process can take an hour or more.
```